### PR TITLE
Mugdha.goel accounts rest endpoints

### PR DESCRIPTION
**Pull request by mgoelswirlds**
This PR has the following:

- api/v1/accounts/{idOrAliasOrEvmAddress} endpoint 
- Fix to change the transaction Id in the transactions api from query parameter to path parameter as in current Rest API.

### CODE REVIEW

File: src/main/java/com/swirlds/recordserver/AccountsService.java [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        JsonObjectBuilder accountBuilder =  getAccountsJsonFields(idParam);
+        if(accountBuilder == null) {
+            response.send(JSON.createObjectBuilder().build());
+            return;
+        }
+        //Read Id from account
+        String accountId = String.valueOf(accountBuilder.build().get("account"));
+        if(!accountId.isBlank()) {
+            var ids = accountId.split(".");
+            accountId = ids.length>1?ids[2]:accountId;
```
Comment (MarcKriguerAtHedera) [inline src/main/java/com/swirlds/recordserver/AccountsService.java]: Sorry, this is just a nit.  I find it much easier to read the "?" and ":" with spaces around each side of each -- it's just a POC, I know, but there's definitely room on the line for some spaces.

**Review by MarcKriguerAtHedera — APPROVED** [review]
One small nit about adding whitespace to a line of code, but this all looks goof to me.

File: src/main/java/com/swirlds/recordserver/AccountsService.java (hunk 1 above)
Comment (mgoelswirlds) [reply src/main/java/com/swirlds/recordserver/AccountsService.java]: Will fix it. Thanks.

Comment (mgoelswirlds) [reply src/main/java/com/swirlds/recordserver/AccountsService.java]: Done.

**Pull request merged**